spec/controllers/lists_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+describe ListsController do
+
+  before {sign_in}
+
+  describe "GET new" do
+    let(:board) {create(:board)}
+    it "renders the new template" do
+      get :new, board_id: board.id
+      expect(response).to render_template(:new)
+    end
+  end
+
+  describe "GET edit" do
+    let(:board) {create(:board)}
```

Comment:
jezeli uzywasz prawie wszedzie tego samego let'a to czemu nie wyniesiesz go wyzej zeby mozna bylo z niego korzysac w kazdym describe. Pamietaj ze jak zrobisz:
```
describe SomeClass
  let(:cos) { create(:cos) }

  describe 'title' do
  # tutaj masz dostep do let'a `cos` ale jak nadpiszesz go np:
  let(:cos) { create(:cos, attribute: 'something') }
  it 'test który korzysta z let'a cos` => ten spec wezmie najblizszego let'a czyli ten z dodatkowym attrybutem
  end
end